lib/tensorflow/tensor.rb

```diff
@@ -121,27 +121,30 @@ def dimension_finder(array)
   #
   def find_type(data)
     start = data if self.rank == 0
-    start = data.flatten[0]  if self.rank != 0 
+    start = data.flatten[0]  if self.rank != 0
```

Comment:
@sebastiandeutsch What do you think of double-spacing between the statement and if?